query-translator/tests/translate.rs

```diff
@@ -888,3 +888,20 @@ fn test_fulltext_inputs() {
                        AND `datoms02`.a = 99");
     assert_eq!(args, vec![make_arg("$v0", "hello"),]);
 }
+
+#[test]
+fn test_instant_range() {
```

Comment:
Can we have another test please testing the `self.mark_known_empty(EmptyBecause::NonInstantArgument);` case?